Containers/talk/start.sh

```diff
@@ -21,9 +21,6 @@ fi
 set -x
 IPv4_ADDRESS_TALK="$(dig nextcloud-aio-talk IN A +short | grep '^[0-9.]\+$' | sort | head -n1)"
 IPv6_ADDRESS_TALK="$(dig nextcloud-aio-talk AAAA +short | grep '^[0-9a-f:]\+$' | sort | head -n1)"
```

Comment:
Are these two always the container IP addresses?

Reply:
yes

Reply:
Gotcha. In that case, I propose additional changes in the upcoming comment.